### PR TITLE
updated vers num and edited class-client.php constructor

### PR DESCRIPTION
**Pull request by richdacuban**
- updated version num 0.1.2 -> 0.1.3
- modified constructor in `class-client.php` as/per @douglas-johnson

### CODE REVIEW

File: inc/class-client.php [hunk 1]
```diff
@@ -28,8 +28,8 @@ class Client {
 	 */
 	public function __construct() {
 
-		$this->content_filter_locations();
-		$this->link_filter_locations();
+		add_action('init', array( $this, 'content_filter_locations'));
```
Comment (douglas-johnson) [inline inc/class-client.php]: @richdacuban nitpick, but can you even out the spacing in these two lines?

- 1 space between `(` and `'init'`
- 1 space between `_locations'` and `)`
- 1 space between `)` and `)`

**Comment by douglas-johnson** [comment]
@richdacuban In the past I've only included the code changes in the pull request. After merging it I would add a single commit to master with the version number changes in the code and the version tag. 

I am not a semantic versioning expert, so if you can do it all in one PR and still do a release with the right version number, go for it.

File: inc/class-client.php (hunk 1 above)
Comment (richdacuban) [reply inc/class-client.php]: @douglas-johnson Apologies, that was messy of me. Edited a/p directions.

**Comment by richdacuban** [comment]
@douglas-johnson Understood, rolled back the version numbers.

**Review by douglas-johnson — APPROVED** [review]

**Comment by douglas-johnson** [comment]
@richdacuban merge and tag at will

**Pull request merged**